Week-4/Exercise-4.1/site.js

```diff
@@ -0,0 +1,25 @@
+const Fibonacci = (number) => {
+  return {
+    [Symbol.iterator]: () => {
```

Comment:
@AakashPandya Looks good to me. Nice work

Reply:
Thanks please approve @RajeshSivanesan